test/run.sh

```diff
@@ -83,19 +83,20 @@ args+=( "-namespace=${NAMESPACE:-default}" )
 args+=( "-timeout=60m" )
 
 if [[ "${GITPOD_REPO_ROOT:-}" != "" ]]; then
-  echo "Running in Gitpod workspace. Fetching USERNAME and USER_TOKEN"
-  USERNAME="$(kubectl --context=dev -n werft get secret integration-test-user -o jsonpath='{.data.username}' | base64 -d)"
+  echo "Running in Gitpod workspace. Fetching USER_NAME and USER_TOKEN"
+  USER_NAME="$(kubectl --context=dev -n werft get secret integration-test-user -o jsonpath='{.data.username}' | base64 -d)"
   USER_TOKEN="$(kubectl --context=dev -n werft get secret integration-test-user -o jsonpath='{.data.token}' | base64 -d)"
+  export USER_NAME
   export USER_TOKEN
```

Comment:
this just makes it easier to copy/paste these four lines and then manually run a test that requires a username+token with `go test`